usage.ftd

```diff
@@ -7,10 +7,10 @@ toc: $ft.toc
 `fpm` CLI uses a few sub-commands to do different things:
 
 - `fpm -h` or `fpm --help` to print help-text and exit.
-- [`fpm build`](/fpm/build/)
-- [`fpm sync`](/fpm/sync/)
-- [`fpm status`](/fpm/status/)
-- [`fpm diff`](/fpm/diff/)
+- [`fpm build`](/build/)
```

Comment:
Remove the slash from the front as well. We put all links without slash in front and slash in end.

Reply:
The slash is still there.

Reply:
I think my language as confusing. We put all links without slash in front. And we always put slash in end.